website/pages/docs/upgrading/instructions/upgrade-to-1-6-x.mdx

```diff
@@ -0,0 +1,228 @@
+---
+layout: docs
+page_title: Upgrading to 1.6.9
+sidebar_title: Upgrading to 1.6.9
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrading to 1.6.9
+
+## Introduction
+
+This guide explains how to best upgrade a multi-datacenter Consul deployment that is using
+a version of Consul >= 1.2.4 and < 1.6.9 while maintaining replication. If you are on a version
+older than 1.2.4, please review our [Upgrading to 1.2.4](/docs/upgrading/instructions/upgrade-to-1-2-x)
+guide. Due to changes to the ACL system, an ACL token migration will need to be performed
+as part of this upgrade. The 1.6.x series is the last series that had support for legacy
+ACL tokens, so this migration _must_ happen before upgrading past the 1.6.x release series.
+Here is some documentation that may prove useful for reference during this upgrade process:
+
+- [ACL System in Legacy Mode](https://www.consul.io/docs/acl/acl-legacy) - You can find
+  information about legacy configuration options and differences between modes here.
+- [Configuration](https://www.consul.io/docs/agent/options) - You can find more details
+  around legacy ACL and new ACL configuration options here. Legacy ACL config options
+  will be listed as deprecates as of 1.4.0.
+
+In this guide, we will be using an example with two datacenters (DCs) and will be
+referring to them as DC1 and DC2. DC1 will be the primary datacenter.
+
+## Requirements
+
+- All Consul servers should be on a version of Consul >= 1.2.4 and < 1.6.9.
+
+## Assumptions
+
+This guide makes the following assumptions:
+
+- You have at least two datacenters configured and have ACL replication enabled. If you are
+  not using multiple datacenters, you can follow along and simply skip the instructions related
+  to replication.
+- You have not already performed the ACL token migration. If you have, please skip all related
+  steps.
+
+## Considerations
+
+There are quite a number of changes between releases. Notable changes
+are called out in our [Specific Version Details](/docs/upgrading/upgrade-specific#consul-1-6-3)
+page. You can find more granular details in the full [changelog](https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#124-november-27-2018).
+Looking through these changes prior to upgrading is highly recommended.
+
+Two very notable items are:
+
+- 1.6.2 introduced more strict JSON decoding. Invalid JSON that was previously ignored might result in errors now (e.g., `Connect: null` in service definitions). See [[GH#6680](https://github.com/hashicorp/consul/pull/6680)].
+- 1.6.3 introduced the [http_max_conns_per_client](https://www.consul.io/docs/agent/options.html#http_max_conns_per_client) limit. This defaults to 200. Prior to this, connections per client were unbounded. [[GH#7159](https://github.com/hashicorp/consul/issues/7159)]
+
+## Procedure
+
```

Comment:
Suggestion: - To make it clear that they should start with the primary datacenter, maybe add a sentence here that in the following instructions, `dc1` is the primary dc